docs/tutorials/standalone/Usage/LSM_single_column_tutorial.jl

```diff
@@ -298,26 +271,34 @@ land = LandHydrology{FT}(;
 # from time `t` to
 # time `t+Δt`, we must compute the infiltration at `t`.
 # This value is stored
-# in `p.soil_infiltration`, and reflects a proper use of the auxiliary
-# or cache state: storing a quantity which we would rather compute once
-# and store, rather than compute twice, once in the soil tendency function,
-# and once in the pond tendency function. This guarantees the same value is
-# used for both equations. In pseudo code, we have:
+# in `p.soil_infiltration`.  In pseudo code, we have:
 # ```julia
 # function make_update_aux(land)
 #          soil_update_aux! = make_update_aux(land.soil)
 #          surface_update_aux! = make_update_aux(land.surface_water)
-#          interactions_update_aux! = make_update_aux(land, land.soil, land.surface_water)
```

Comment:
do we need to replace this with `initialize_additional_lsm_aux`?

Reply:
fixed in PR#417!